README.md

```diff
@@ -11,81 +11,27 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
 - Connect Helm-based applications to external services with Service Catalog Bindings
 - Secure authentication and authorization based on Kubernetes [Role-Based Access Control](docs/user/access-control.md)
 
-## Quickstart
+## Quick Start
 
-Kubeapps assumes a working Kubernetes cluster (v1.8+), [`Helm`](https://helm.sh/) (2.10.0+) installed in your cluster and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE),`minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
-
-> On GKE, you must either be an "Owner" or have the "Container Engine Admin" role in order to install Kubeapps.
-
-> **IMPORTANT**: Kubeapps v1.0.0-alpha.4 and below used the `kubeapps` CLI to install Kubeapps, Tiller and other components. Please [see the migration guide](docs/user/migrating-to-v1.0.0-alpha.5.md) when upgrading from a previous version to v1.0.0-alpha.5 and above.
-
-Use the Helm chart to install the latest version of Kubeapps:
+The easiest way of setting up Kubeapps is trough its Helm chart:
```

Comment:
This sounds to me like there's another option, maybe we should just say:

```suggestion
Use Helm to install Kubeapps in your cluster: 
```

?

Reply:
or actually the current text "Use the Helm chart to install the latest version of Kubeapps:" is pretty good too